tests/README.md

```diff
@@ -1,5 +1,9 @@
 To use these tests run the following:
 
+The environment variables set by the pipeline need to be overwritten. The pipeline automatically overwrites the target host, therefore we need to unset them so we can deploy the pods within the correct environment. 
+
 ```
-ansible-playbook ./openshift-deployment-ansible/tests/all.yml --extra-vars "hostIP=<IP_address_of_Bastion_host>"
+for var in \$(export | grep KUB | awk '{ print \$2 }' | sed 's/\\=.*//g'); do unset \$var; done; source ./openstack_rc.sh ;
+ansible-playbook ./openshift-deployment-ansible/tests/all.yml --extra-vars "hostIP=<IP_address_of_Bastion_host>" --extra-vars OPENSHIFT_USERNAME=\"\$OPENSHIFT_USERNAME\" --extra-vars OPENSHIFT_PASSWORD=\"\$OPENSHIFT_PASSWORD\" --extra-vars server=\"<target_server>\"
```

Comment:
Make this usage information about the playbook - this is too specific about running in the pipeline again and not very readable. It should be easy to understand. Take the <IP_address_of_Bastion_host> as an example.